### PR TITLE
{devel}[intel/2017a] pbs_python v4.6.0 w/ Python-2.7.13

### PR DESCRIPTION
**Pull request by shailr**
Added a new easyconf for pbs_python with updated Intel compiler. Other pbs_python configs fail with newer RHEL/CentOS updates.

Refer this:
https://software.intel.com/en-us/articles/intel-compiler-version-16-not-compatible-with-recent-libcso6

### CODE REVIEW

File: easybuild/easyconfigs/p/pbs_python/pbs_python-4.6.0-intel-2017a-Python-2.7.13.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+dependencies = [('Python', '2.7.13')]
+
+osdependencies = ['torque-devel']
+
+# this is needed for setup.py script
+configopts = "--datarootdir=%(installdir)s --docdir=%(installdir)s"
+prebuildopts = 'export CFLAGS="$CFLAGS -I/usr/include/torque/" && '
+preinstallopts = prebuildopts
+
+pythonpath = 'lib/python%(pyshortver)s/site-packages'
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/pbs_python/pbs_python-4.6.0-intel-2017a-Python-2.7.13.eb]: Due to recent changes in EasyBuild the "pythonpath" variable need to be renamed to local_pythonpath.

File: easybuild/easyconfigs/p/pbs_python/pbs_python-4.6.0-intel-2017a-Python-2.7.13.eb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+description = """The pbs_python package is a wrapper class for the Torque C library. With this package you now can
+ write utilities/extensions in Python instead of C. We developed this package because we want to replace xpbsmon by
+ an ascii version named pbsmon. PBSQuery is also included in this package. This is a python module build on top of
+ the pbs python module to simplify querying the batch server, eg: how many jobs, how many nodes, ..."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+source_urls = ['http://ftp.sara.nl/pub/outgoing/']
+sources = [SOURCE_TAR_GZ]
+
```
Comment (akesandgren) [inline easybuild/easyconfigs/p/pbs_python/pbs_python-4.6.0-intel-2017a-Python-2.7.13.eb]: Please add a checksum, eb --inject-checksum is your friend.

**Review by akesandgren — APPROVED** [review]

**Comment by akesandgren** [comment]
I don't have torque-devel available so someone else need to make a test report on this.
@shailr Can you do it?
eb --from-pr 6171 -u

**Comment by boegel** [comment]
To be honest, I don't think it's still worth merging this...

But maybe @lexming can submit a test report for this? They're still using Torque...

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node101.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v2 @ 2.80GHz, Python 2.7.5
See https://gist.github.com/a35bf0b8cfe6c84b7b9702d11ad1233e for a full test report.

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @shailr!

**Pull request merged**